content/php/concepts/string-functions/string-functions.md

```diff
@@ -0,0 +1,19 @@
+---
+Title: 'String Functions'
+Description: 'PHP has several built-in functions for manipulating strings.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Design'
+  - 'Web Development'
+Tags:
+  - 'Strings'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/front-end-engineer-career-path'
+  - 'paths/computer-science'
```

Comment:
```suggestion
  - 'learn-php'
  - 'paths/computer-science'
```

Isn't PHP a backend-only language? Either way, we should stick with two (2) links, per the [catalog content documentation](https://github.com/Codecademy/docs/blob/main/documentation/catalog-content.md#php)